web-server/public/js/prosumer-data.js

```diff
@@ -0,0 +1,264 @@
+var exitedPage = false;
```

Comment:
This is an old file that was replaced by a cleaner and better one. Remove this file.

Reply:
Ok!